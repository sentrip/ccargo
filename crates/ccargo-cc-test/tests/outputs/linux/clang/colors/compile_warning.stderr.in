[1mmain.c:4:10: [0m[0;1;35mwarning: [0m[1mincompatible pointer types initializing 'int *' with an expression of type 'long long *' [-Wincompatible-pointer-types][0m
    int* x = &v;
[0;1;32m         ^   ~~
[0m[1mmain.c:5:10: [0m[0;1;35mwarning: [0m[1mincompatible pointer types initializing 'int *' with an expression of type 'long long *' [-Wincompatible-pointer-types][0m
    int* y = &v;
[0;1;32m         ^   ~~
[0m2 warnings generated.
