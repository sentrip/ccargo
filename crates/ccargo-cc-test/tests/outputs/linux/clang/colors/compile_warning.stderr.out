[0m[1m[33mwarning[0m[1m: [0m[0m[1mincompatible pointer types initializing 'int *' with an expression of type 'long long *' [-Wincompatible-pointer-types][0m[0m[1m[36m
   --> [0mmain.c:4:10[0m
    4 |     int* x = &v;[0m
      | [0;1;33m         ^   ~~[0m
[0m[1m[33mwarning[0m[1m: [0m[0m[1mincompatible pointer types initializing 'int *' with an expression of type 'long long *' [-Wincompatible-pointer-types][0m[0m[1m[36m
   --> [0mmain.c:5:10[0m
    5 |     int* y = &v;[0m
      | [0;1;33m         ^   ~~[0m
[0m2 warnings generated.[0m
